.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 yarn.lock
 coverage
 .nyc_output
+.vscode
```

Comment:
This should be in your own global gitignore. The project gitignore is only for project-related ignores. https://gist.github.com/subfuzion/db7f57fff2fb6998a16c